.php_cs.dist

```diff
@@ -47,7 +47,7 @@ return PhpCsFixer\Config::create()
         'phpdoc_to_comment' => true,
         'phpdoc_trim' => true,
         'single_quote' => true,
-        'ternary_to_null_coalescing' => true,
+        'ternary_to_null_coalescing' => false,
```

Comment:
Good catch! This will keep compatibility with older PHP versions that don't support the coalescing operator. 👍